### ISSUE TITLE
Roomify and Google Maps > Upgrade from V2 to V3

### ISSUE THREAD

**Issue by ghost** [Roomify/RfA#394]
## Expected Behavior

Google maps should be updated toward the V3 instead of the V2 version, seeing as you now need an a
## Current Behavior

Google maps gives an NoApiKeys error in the Console, and all Location modules don't work due to this issue.

--
## Steps to Reproduce from a clean installation of the latest release

1. Create a new property and in the Location tab, set a place using the Google Map
2. Add a valid Google Javascript Maps API Key
3. Look at the map as a user
4. Look at the Dev console (f12), it should give an error even if the key is valid

## Context (Environment)

I am not able to remove the map feature nor able to use it, my customers also see the useless map and it doesn't look professional as it used to be

Windows 7 + Chrome Version 67.0.3396.99 (Official build) (32-bits)

**Comment by blackdoginet** [comment]
Same problem here. Locations mapping is not displaying Google Maps correctly. 
Get directions config shows map OK.
Add new location does not. Also home page map is not displaying correctly. No API key error in Console
![screenshot from 2018-10-16 13-16-39](https://user-images.githubusercontent.com/17192852/47015779-ca31a580-d145-11e8-8cac-950d395c4b87.png)
![screenshot from 2018-10-16 13-15-18](https://user-images.githubusercontent.com/17192852/47015780-ca31a580-d145-11e8-913d-b40083241b3d.png)

**Comment by acrollet** [comment]
Do you have a billing account associated with your API key?

**Comment by blackdoginet** [comment]
Thanks for rapid response
It is the Google API key that is not being submitted correctly. I am running self hosted Roomify. (Default version 1.5.4)  The Google key is unrestricted and has nowhere near it limit of hits.
Hope this helps

**Comment by acrollet** [comment]
So you do have a billing account associated with the key? Google now requires this even for free usage.

**Comment by blackdoginet** [comment]
Yes and same key works on other sites using Joomla (PHP) and Odoo (Python)
Joomla
![screenshot from 2018-10-16 14-05-23](https://user-images.githubusercontent.com/17192852/47018403-9312c280-d14c-11e8-926d-0bcdea6377b1.png)

Roomify with dev console
![screenshot from 2018-10-16 14-04-34](https://user-images.githubusercontent.com/17192852/47018404-9312c280-d14c-11e8-8981-ddc6a7652c54.png)

**Comment by blackdoginet** [comment]
Any updates please?
